### ISSUE TITLE
Suggestion: Donationlink

### ISSUE THREAD

**Issue by enterih** [NoCheatPlus/Issues#507]
#### Short description of the issue.
I wanna donate but no links or button are exist

#### How to reproduce the issue (is it random / always happens / side conditions)?
Always happen

#### Have you made any (noteworthy) changes to the default configuration of NCP?
No

#### Do you run any other special plugins that affect game mechanics next to NC+ (adding skills, machines, adding/changing blocks, other anti cheating)?
No

**Comment by asofold** [comment]
I'm making another attempt to get such working. However there are two things i need to be sure about:
1. German tax law (very complicated and unfriendly towards receiving extra money with the "Neuland").
2. Privacy (perhaps a paypal business account  does it).

**Comment by enterih** [comment]
Does a paypal me - link not handle it?

Am 14.01.2018 um 23:03 schrieb asofold <notifications@github.com<mailto:notifications@github.com>>:


I'm making another attempt to get such working. However there are two things i need to be sure about:

  1.  German tax law.
  2.  Privacy (perhaps a paypal business account does it).

**Comment by asofold** [comment]
Not sure, the idea is the my name is not shown in the transaction by default. While it may be more interesting to know whom one is sending money to, it also changes how things are set up here (so far we did with few/none of names).

**Comment by 0-x-2-2** [comment]
you can indeed hide your name and show a "business" name with a paypal business account but you could accept cryptocurrency if you are really worried about privacy.

**Comment by asofold** [comment]
@w-o-a-h Yes, though crypto currencies may be even more difficult to maintain tax-wise and might also need constant extra nursing for e-something-wallets and e-something-exploits.

Probably i'm not intending to go all out on that. Despite paypal not being 100% great either, it's a widely accepted thing, and i might also intend to do some other donation based stuff in future, so the business account direction may be one of the closest matches for a not too complicated approach, keeping a minimum distance, but also looking (not too far) ahead.

I'll take legal advice on how to approach things with software in germany - the new realms of technology seem not to have been used to enable people to earn something extra on a potentially irregular basis yet, instead more modern forms of slavery have been reintroduced/invited first. Getting sorted into random business types or forced out/in jobs/something isn't what i'm looking forward to (not only considering NCP), so i'll want to have odds and ends covered somehow.

**Comment by asofold** [comment]
Looks like i can more or less safely start off, so unless paypal has objections or complicates things beyond reasonable, i'll be able to open up "donations" somehow more or less soon. Of course i'll not implement something certain, just because a million $ transfer has 'implement colored sheep' on it :) - and it's of course not donations law/tax-wise. Depending on what happens and how this works tax-wise, i might at some point offer something like donation goals/barriers linked to the implementation of (rather complex) features.

So let's see what happens ... closing once things are up, could take another week.
